src/backends/interflop-vprec/Makefile.am

```diff
@@ -4,6 +4,7 @@ libinterflop_vprec_la_CFLAGS = -DBACKEND_HEADER="interflop_vprec"
 if WALL_CFLAGS
 libinterflop_vprec_la_CFLAGS += -Wall -Wextra -Wno-varargs
 endif
+libinterflop_vprec_la_CFLAGS += -march=native
```

Comment:
I need the flag `-march=native` for all backends which use vector interface else it will not work because the compiler can't match vector types with expected vector set instruction.